### PR TITLE
Update Constants file to reflect v0.01

### PR DESCRIPTION
**Pull request by dangerousfood**
Updated constants to reflect the changes that were made in the spec version v0.01.

### CODE REVIEW

File: ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/Constants.java [hunk 1]
```diff
@@ -16,168 +16,79 @@
...
-
-  public static final long GENESIS_SLOT = (long) Math.pow(2, 19);
-  public static final long GENESIS_EPOCH = GENESIS_SLOT / Constants.EPOCH_LENGTH;
+  public static final int DOMAIN_DEPOSIT = 0; //
+  public static final int DOMAIN_ATTESTATION = 1; //
+  public static final int DOMAIN_PROPOSAL = 2; //
+  public static final int DOMAIN_EXIT = 3; //
+  public static final int MAX_DOMAIN_RANDAOEXITS = 4; //
 
-  public static String getConstantsAsString() {
```
Comment (akhila-raju) [inline ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/Constants.java:94]: Do we want this removed? What's the reasoning for removal?

File: ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/BeaconState.java [hunk 2]
```diff
@@ -172,46 +170,51 @@ public BeaconState get_initial_beacon_state(
...
-            new ArrayList<>(),
-            new ArrayList<>(),
-            new ArrayList<>(),
-
-            // Ethereum 1.0 chain data
-            latest_eth1_data,
-            new ArrayList<>());
+    // todo after update v0.01 constants no longer exist
+    BeaconState state = new BeaconState();
+    //    BeaconState state =
```
Comment (akhila-raju) [inline ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/BeaconState.java:178]: Why is this commented out, rather than removed?

**Review by akhila-raju — COMMENTED** [review]
A few comments :)

File: ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/Constants.java (hunk 1 above)
Comment (dangerousfood) [reply ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/Constants.java:94]: We can add it back later. getConstantsAsString cannot be generated using the code generation tools.

File: ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/BeaconState.java (hunk 2 above)
Comment (dangerousfood) [reply ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/BeaconState.java:178]: Too many cascading changes when modifying the BeaconState constructor.

**Review by akhila-raju — APPROVED** [review]

**Pull request merged**